src/canopy_canvars_mod.F90

```diff
@@ -52,6 +52,18 @@ MODULE canopy_canvars_mod
     real(rk), allocatable :: ppfd_shade240_tmp   ( : , :, : )          ! PPFD for shaded leaves (umol phot/m2 s)
     real(rk), allocatable :: tmp2mref_tmp        ( : , : )          ! 2-meter (AGL) input reference air temperature (K)
     real(rk), allocatable :: ubzref_tmp          ( : , : )          ! 10-meter (AGL) input reference wind speed (m/s)
+    real(rk), allocatable :: tleaf_sun24_rst     ( : , :, : )          ! Leaf temp for sunlit leaves (K)
```

Comment:
Similar comment here and below for having new "rst" variables. Are these really needed>?